docs/mounting.md

```diff
@@ -22,6 +22,10 @@ Console:
 
     GOOGLE_APPLICATION_CREDENTIALS=/path/to/key.json gcsfuse [...]
 
+On OS X and Linux you can also use fstab option in `/etc/fstab` system file:
```

Comment:
Well, good point. My comment was based on the existing last note of this file though:

```
On both OS X and Linux, you can also add entries to your /etc/fstab file like the following:
```